pandas/tests/frame/test_indexing.py

```diff
@@ -1944,6 +1944,29 @@ def test_set_value_resize(self, float_frame):
         with pytest.raises(ValueError, match=msg):
             res3._set_value("foobar", "baz", "sam")
 
+    def test_loc_indexing_preserves_index_category_dtype(self):
```

Comment:
can you move this to: TestDataFrameIndexingCategorical

Reply:
Thanks, @jreback - unfortunately, I couldn't find `tests/frame/indexing/test_...`, that's why I put the test in `tests/frame/test_indexing.py`.

However, there seems to be a slight inconsistency in the `tests/frame/` and `tests/series/` structure as there's a `indexing` subfolder in `series` but not in `frame`.

Reply:
this is a class name *in* the current file

> However, there seems to be a slight inconsistency in the tests/frame/ and tests/series/ structure as there's a indexing subfolder in series but not in frame.

yes I know, there is an issue about fixing this (happy to have a PR), but orthogonal to this one.

Reply:
Ah, I see - moved the test there. Thanks